tools/schedule-migration/main.go

```diff
@@ -23,7 +23,7 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-type Item struct {
+type Object struct {
```

Comment:
just a little curious, using `Item` as the strut name is a bad smell? or is there revive warnings on here? 

Or it just renames because easy to understand?

Reply:
rest LGTM

Reply:
> Or it just renames because easy to understand?

Yep. I think it would be more appropriate to use `Object`. No warnings here.

The bad smell that occurred in this file is using `new` as the variable name. At this line:

https://github.com/chaos-mesh/chaos-mesh/blob/40080be801e7ea9230813b0ec971e6292c74c278/tools/schedule-migration/main.go#L45

The `new` is a reserved keyword.

Reply:
got that